fiftyone/operators/panel.py

```diff
@@ -1,62 +1,79 @@
 """
-FiftyOne operators.
+FiftyOne panels.
 
 | Copyright 2017-2024, Voxel51, Inc.
 | `voxel51.com <https://voxel51.com/>`_
 |
 """
 
+import pydash
+
 import fiftyone.operators.types as types
 from fiftyone.operators.operator import OperatorConfig, Operator
 
-import pydash
-
 
 class PanelConfig(OperatorConfig):
-    """A configuration for a panel operator."""
+    """Configuration for a panel.
+
+    Args:
+        name: the name of the panel
+        label: the display name for the panel
+        icon (None): the icon to show in the panel's tab
+        light_icon (None): the icon to show in the panel's tab when the App is
+            in light mode
+        dark_icon (None): the icon to show in the panel's tab when the App is
+            in dark mode
+        allow_multiple (False): whether to allow multiple instances of the
+            panel to be opened
+    """
 
     def __init__(
         self,
         name,
         label,
         icon=None,
-        dark_icon=None,
         light_icon=None,
+        dark_icon=None,
         allow_multiple=False,
         **kwargs
     ):
         super().__init__(name)
         self.name = name
         self.label = label
         self.icon = icon
-        self.dark_icon = dark_icon
         self.light_icon = light_icon
+        self.dark_icon = dark_icon
         self.allow_multiple = allow_multiple
         self.unlisted = True
         self.on_startup = True
         self.kwargs = kwargs  # unused, placeholder for future extensibility
 
     def to_json(self):
-        d = super().to_json()
         return {
-            **d,
             "name": self.name,
             "label": self.label,
             "icon": self.icon,
-            "dark_icon": self.dark_icon,
             "light_icon": self.light_icon,
+            "dark_icon": self.dark_icon,
             "allow_multiple": self.allow_multiple,
         }
 
 
-# Alias for backwards compatibility
```

Comment:
@ritch just confirming that I'm deleting `PanelOperatorConfig` here as discussed